tests/ui/derive-invariant/helper-wrong-expr/expected

```diff
@@ -0,0 +1,9 @@
+error[E0308]: mismatched types
+   |
+14 |     #[safety_constraint(x >= 0)]
```

Comment:
Can you please remove the line number of expected files. It makes any change to the file cumbersome, even adding comments.

Reply:
Removed.